spring-integration-mongodb/src/test/java/org/springframework/integration/mongodb/config/MongoDbOutboundGatewayParserTests.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.mongodb.config;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import org.springframework.beans.factory.parsing.BeanDefinitionParsingException;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.springframework.expression.common.LiteralExpression;
+import org.springframework.expression.spel.standard.SpelExpression;
+import org.springframework.integration.mongodb.outbound.MongoDbOutboundGateway;
+import org.springframework.integration.test.util.TestUtils;
+
+/**
+ * @author Xavier Padró
+ * @since 5.0
+ */
+public class MongoDbOutboundGatewayParserTests {
+
+	@Test
+	public void minimalConfig() {
+		ClassPathXmlApplicationContext context =
+				new ClassPathXmlApplicationContext("outbound-gateway-parser-config.xml", this.getClass());
```

Comment:
I think we can just go with the 
```
@ContextConfiguration
@RunWith(SpringRunner.class)
@DirtiesContext
```
and avoid extra loads for the the same config in each test method.
I that case the `outbound-gateway-parser-config.xml` should be renamed to the `MongoDbOutboundGatewayParserTests-context.xml`